### PR TITLE
feat(ResliceCursorWidget): Add possibility to user to manually set center

### PR DESCRIPTION
**Pull request by laurennlam**
@floryst @finetjul

### CODE REVIEW

File: Sources/Widgets/Widgets3D/ResliceCursorWidget/api.md [hunk 1]
```diff
@@ -36,4 +38,21 @@ These sub states contain :
...
+If you want to add the previous feature, then you'll have to defined the
+```
+renderer[axis].widgetInstance.onWidgetChange(() => {
+		renderer
+        // No need to update plane nor refresh when interaction
+        // is on current view. Plane can't be changed with interaction on current
+        // view. Refreshs happen automatically with `animation`.
+        .filter((_, index) => index !== axis)
+        .forEach((viewer) => {
+					viewer.updateReslice();
```
Comment (finetjul) [inline Sources/Widgets/Widgets3D/ResliceCursorWidget/api.md:55]: indent is incorrect.

updateReslice() does not exist

**Comment by finetjul** [comment]
LGTM.
Please rebase

**Comment by floryst** [comment]
LGTM. I'm re-triggering the checks, since the failure is external to the PR.

**Comment by floryst** [comment]
And we'll need another rebase, since master got updated about an hour ago.

**Pull request merged**